specreduce/tracing.py

```diff
@@ -50,7 +50,8 @@ def shift(self, delta):
         delta : float
             Shift to be applied to the trace
         """
-        self.trace += delta
+        # act on self.trace.data to ignore the mask and then re-mask when calling _bound_trace
+        self.trace = self.trace.data + delta
```

Comment:
this seems to work locally for me, but on CI is giving `TypeError: unsupported operand type(s) for +: 'memoryview' and 'int'`.

Reply:
("fixed" by casting with `np.asarray`)